apps/blockchain-monitor/src/monitors.ts

```diff
@@ -0,0 +1,8 @@
+import { MonitorSchedule } from '@notional-finance/monitors';
+import { MonitorJob, exchangeRateMonitor } from '@notional-finance/monitors';
+
+export const monitors: Map<string, MonitorJob[]> = new Map([
+  [MonitorSchedule.EVERY_MINUTE, [exchangeRateMonitor]],
```

Comment:
This is where you add a new job, just add it to the array for the schedule you want.